packages/jade-coffee/package.js

```diff
@@ -1,19 +1,21 @@
 Package.describe({
   summary: "Jade template language with inline coffeescript and Javascript support",
-  version: "0.1.4",
+  version: "0.1.6",
   name: "xiphy:jade-coffee",
   git: "https://github.com/xiphias/meteor-jade-coffee.git",
   documentation: "../../README.md"
 });
 
 Package.registerBuildPlugin({
-  name: "compileJade",
+  name: "compileJadeBatch",
```

Comment:
This doesn't work probably as there's no compileJadeBatch function yet, as we're using the old handler.js. The build shouldn't pass for this.